prac_08/box_layout.kv

```diff
@@ -0,0 +1,15 @@
+BoxLayout:
+    orientation: 'vertical'
+    Button:
+        text: 'Clear'
+        on_press: app.handle_clear()
+    TextInput:
+        id: input_name
+        text: ''
+    Button:
+        text: 'Greet'
+        on_press: app.handle_greet()
+    Label:
+        text: 'Enter your name'
+        color: (1, 1, 0, 1)  # Yellow color in RGBA
+        id: output_label
```

Comment:
I think for readability purposes maybe put the id at the top.
It'll make the code easier to understand if you know what the code is for.